ehr2vec/configs/p1_cluster_backup/05_01_simulate_binary_outcome.yaml

```
@@ -1,18 +1,18 @@
-  model_path: /dtu/p1/kirkle/corebehrt_phair/pretrain/run_01/finetune_EXPOSURE_censored_1_hours_pre_EXPOSURE_followup_start_at_index_date_ft_exposure
-  run_name: simulate_binary_outc
-  output: /dtu/p1/kirkle/corebehrt_phair/features/synthea/100k/outcomes/EXPOSURE_simulated_outcome/
-
+  model_path: /dtu/p1/kirkle/PHAIR_project/corebehrt_phair/pretrain/run_02/finetune_amlodipine_80_censored_30_days_pre_amlodipine_80_followup_start_at_index_date_run_00
+  run_name: simulate_binary_outcome
+  output_path: /dtu/p1/kirkle/PHAIR_project/corebehrt_phair/features/synthea_0712/100k/outcomes/amlodipine_random_80/simulated_outcome_a_4p0_b_0p0/
+predictions_file: predictions_and_targets_calibrated_isotonic.csv
 simulation:
-  _target_: ehr2vec.simulation.binary_outcome.tbehrt
+  _target_: ehr2vec.simulation.binary_outcome.tbehrt # a * exposure + b * (ps + c)
   params: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)